vendor/assets/stylesheets/fenrir/building_blocks/mixins/_wrappers.scss

```diff
@@ -0,0 +1,126 @@
+
+
+
+//----------------------------------------------\\
+// Charlie slice wrapper
+//----------------------------------------------\\
+
+@mixin w-slice() {
+  padding-left: 15px;
+  padding-right: 15px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+
+  &--no-wrap {
+    flex-wrap: nowrap;
+  }
+
+  &--no-padding {
+    padding: 0;
+  }
+}
+
+
+
+//----------------------------------------------\\
+// Charlie card wrapper
+//----------------------------------------------\\
+
+@mixin w-card() {
+  position: relative;
+  width: 100%;
+  border-radius: $br;
+  margin: 15px 0;
+  box-shadow: 0 0 0 1px color('black', A5);
```

Comment:
It's a tricky one, but I think the way you described, where inclusion and the specificity overriding the property makes sense.